job-service-contract/src/main/resources/com/github/jobservice/contract/swagger.yaml

```diff
@@ -90,6 +90,18 @@ paths:
           - Completed
           - Inactive
           - NotFinished
+      - name: status
+        in: query
+        type: string
+        required: false
+        description: Active - only those with active status will be returned; Cancelled - only those with cancelled status will be returned; Completed - only those results with completed status will be returned; Failed - only those results with failed status will be returned; Paused - only those results with paused status will be returned, Waiting - only those results with waiting status will be returned.
+        enum:
+          - Active
+          - Cancelled
+          - Completed
+          - Failed
+          - Paused
+          - Waiting
```

Comment:
From reading the ticket it seems like the expectation was that this parameter would be added to the `/jobStats/count` endpoint as well.  Was this just missed?  Can it be easily added or should we extract it to a separate ticket?

Reply:
This was missed but should be easy to add.